xbmc/video/VideoReferenceClock.cpp

```diff
@@ -161,6 +174,9 @@ void CVideoReferenceClock::Process()
       delete m_pVideoSync;
       m_pVideoSync = NULL;
     }
+    // Try glx fallback
+    if (glx_fallback)
```

Comment:
that is error prone. we already have an exit criteria which is SetupSucess and there is already a test for glx_fallback some lines above.